src/ign.cc

```diff
@@ -129,9 +130,11 @@ extern "C" IGNITION_GAZEBO_VISIBLE int runServer(const char *_sdfString,
   }
   cmpPath += ".zip";
 
+  std::cout << "Record Topic[" << _recordTopics << std::endl;
```

Comment:
Can this be `ignmsg`?

```suggestion
  ignmsg << "Record Topic[" << _recordTopics << std::endl;
```

Reply:
Removed this testing cout in ffe5749c73099296b491b28a6a3fa61ff291974e